test/fixtures/HTML/no-content.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<html>
+<meta name="generator" value="Some sick tool nobody's using yet" />
```

Comment:
> Some sick tool

You can tell an Australian wrote this. 😁 Didn't even need `<html lang="en-AU">`.